httpdocs/js/notary.js

```diff
@@ -95,14 +105,16 @@ window.onload = function() {
     fieldset.setAttribute('disabled', '');
     const searchCitizen = event.currentTarget;
     searchCitizen.classList.add('is-loading');
-    const familyName = document.getElementById('family-name').value;
-    const givenNames = document.getElementById('given-names').value;
-    judge = document.getElementById('judge').value;
-    let parameters = `latitude=${latitude}&longitude=${longitude}&radius=${radius}&judge=https://${judge}`;
+    const familyName = sanitizeString(document.getElementById('family-name').value);
+    const givenNames = sanitizeString(document.getElementById('given-names').value);
+    judge = sanitizeString(encodeURIComponent(sanitizeString(document.getElementById('judge').value)));
```

Comment:
That seems overkill:
```suggestion
    judge = sanitizeString(document.getElementById('judge').value);
```